GNUmakefile

```diff
@@ -57,7 +57,6 @@ test-compile:
 	go test -c $(TEST) $(TESTARGS)
 
 lint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
```

Comment:
why?

Reply:
I think there is no need to call this command every time to run linter.

Reply:
we need it to have fixed version of linter. Last versions could be broken, it's unsafe.